src/main/java/net/datafaker/providers/base/Image.java

```diff
@@ -3,97 +3,193 @@
 
 import javax.imageio.ImageIO;
 import java.awt.Color;
-import java.awt.Graphics2D;
+import java.awt.*;
 import java.awt.image.BufferedImage;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.util.Base64;
 
 import static java.awt.Color.WHITE;
+import static net.datafaker.providers.base.Image.ImageType.BMP;
+import static net.datafaker.providers.base.Image.ImageType.GIF;
+import static net.datafaker.providers.base.Image.ImageType.JPEG;
+import static net.datafaker.providers.base.Image.ImageType.PNG;
+import static net.datafaker.providers.base.Image.ImageType.SVG;
+import static net.datafaker.providers.base.Image.ImageType.TIFF;
 
 /**
- * Generates base64 encoded PNG, GIF, JPG and SVG images.
+ * Generates base64 encoded raster and vector images.
  *
  * @since 2.3.0
  */
 public class Image extends AbstractProvider<BaseProviders> {
 
-    private static final int WIDTH = 256;
-    private static final int HEIGHT = 256;
-    private static final int BOX_SIZE = WIDTH / 8;
+    private static final int DEFAULT_WIDTH = 256;
+    private static final int DEFAULT_HEIGHT = 256;
+
+    public enum ImageType {
+        // See https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+        BMP("image/bmp"),
+        GIF("image/gif"),
+        JPEG("image/jpeg"),
+        PNG("image/png"),
+        SVG("image/svg+xml"),
+        TIFF("image/tiff");
+
+        private final String mimeType;
+
+        ImageType(String mimeType) {
+            this.mimeType = mimeType;
```

Comment:
@kingthorin what do you mean exactly? There's a unit test here which I believe does exactly that: 

https://github.com/datafaker-net/datafaker/pull/1305/files/eae5243a85ece3003ca3dbe4087c35b2925bbdd3#diff-06ae773f58d0da4e8eea108c7fc94f3afb8e9fc5112cead2597755fdcf1386d7R49

It checks if the image type is really supported by checking the Base64 image contents. If it's a type which is not supported by the JVM, such as APNG or WEBP, this would fails. If you'd make a change to any of the mime types, the test would fail. Making it private is redundant for enums. 

So, what did I miss? ;-)

Reply:
I meant that it would match the enum properly

Reply:
I guess I need to see "how" it fails. I commented here before reading on. Even with the test that you're mentioning it isn't clear to me if it fails somewhat silently or if the dev gets something useful.

Reply:
I'm not really sure I follow. How can you fail this code? It's all private code, you can't call this constructor.